data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -2603,6 +2603,45 @@ spec:
                           type: string
                         type: array
                     type: object
+                  eipAllocations:
+                    description: |-
+                      eipAllocations contains Elastic IP (EIP) allocations for AWS resources
+                      within the cluster.
+                    properties:
+                      ingressNetworkLoadBalancer:
```

Comment:
This is aligned with an old discussion in the EP (that disappeared[1]?!), I added a note in the EP[2] to re-inforce, looks like this change is as expected allowing future expansion if needs.

[1] https://github.com/openshift/enhancements/pull/1688/files#r1845359952